.ddqa/config.toml

```diff
@@ -18,13 +18,21 @@ github_team = "agent-processing-and-routing"
 github_labels = ["team/agent-processing-and-routing"]
 exclude_members = []
 
-[teams."Agent Shared Components"]
-jira_project = "ASCII"
+[teams."Agent Runtimes"]
+jira_project = "ARUN"
 jira_issue_type = "QA"
 jira_statuses = ["To Do", "In Progress", "Done"]
-github_team = "agent-shared-components"
-github_labels = ["team/agent-shared-components"]
-exclude_members = ["sgnn7", "truthbk", "cmourot"]
+github_team = "agent-runtimes"
+github_labels = ["team/agent-runtimes"]
+exclude_members = ["sgnn7", "cmourot"]
+
+[teams."Agent Configuration"]
+jira_project = "ACFG"
+jira_issue_type = "QA"
```

Comment:
These `QA` issue types are custom and need to be recreated. We may want to slice this out into a separate PR when. that's fixed up or change these to `Task` for now.